platform/ios/src/MGLMapboxEvents.m

```diff
@@ -8,6 +8,8 @@
 static NSString * const MGLMapboxMetricsDebugLoggingEnabled = @"MGLMapboxMetricsDebugLoggingEnabled";
 static NSString * const MGLTelemetryAccessToken = @"MGLTelemetryAccessToken";
 static NSString * const MGLTelemetryBaseURL = @"MGLTelemetryBaseURL";
+static NSString * const kMMEEventsProfile = @"MMEEventsProfile";
```

Comment:
Let’s make the naming for these consistent with the other constants (e.g., remove the `k` Hungarian notation, use a consistent prefix, etc.).